_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
Can you also add in description that it uses min and max cell voltage to calculate the scaling for the gain